hw16/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "devDependencies": {
+      "@types/mocha": "^9.1.1"
+    },
+    "dependencies": {
+      "concatenator": "^0.0.1",
+      "prettier": "^2.7.1",
+      "save-dev": "^0.0.1-security",
+      "ts-node": "^10.9.1"
```

Comment:
Please delete packages, that you don't use so far: `concatenator`, `save-dev`, and so on.